content/en/blog/2023/native-sidecars/index.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Kubernetes Native Sidecars in Istio
+description: Demoing the new SidecarContainers feature with Istio.
+publishdate: 2023-07-25
+attribution: "John Howard (Google)"
+keywords: [istio,sidecars,kubernetes]
+---
+
+From the beginning of Istio, applications have been augmented with service mesh functionality by using the "sidecar" pattern:
```

Comment:
nit, it is a good practice to tell what this blog post about in a first sentence. So snippets are better extracted.  Something along the lines:

From the beginning of Istio... sidecar pattern. Kubernetes is winning new workloads and Istio hits limitations of sidecar implementation. With 1.28 and native sidecar support, Istio can be used safely and more reliably on these new workloads.

Reply:
I would go with something like

"If you have heard anything about service meshes, it is that they work using the _sidecar pattern_: a proxy server is deployed alongside your application code.  The sidecar pattern is just that: a pattern; up until this point, there has been no formal support for 'sidecar containers' in Kubernetes at all.  This has caused no end of problems: what if you have a job that terminates by design, but a sidecar container that doesn't?  This exact use case is the [most upvoted issue ever on the Kubernetes GitHub](https://github.com/kubernetes/kubernetes/issues/25908).

A formal proposal for adding sidecar support in Kubernetes was raised in 2019, with many stops and starts along the way. After a reboot of the project last year, formal support for sidecars is being released to Alpha in Kubernetes 1.28. Istio has implemented support for this feature, and in this post you can learn how to update your configuration to take advantage of it."

Reply:
I lifted most of Craig's comment - thanks!